elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -1,112 +1,90 @@
 /* eslint-env jquery, browser */
 
-const TABWIDTH = 50; // defines how many px the answer block is indented by, when the student
-// drags and indents a block
-var MAX_INDENT = 4; // defines the maximum number of times an answer block can be indented
+window.PLOrderBlocks = function (uuid, options) {
+  const TABWIDTH = 50; // defines how many px the answer block is indented by, when the student
+  // drags and indents a block
+  let maxIndent = options.maxIndent; // defines the maximum number of times an answer block can be indented
 
-function set_max_indent(event) {
-  MAX_INDENT = event.target.getAttribute('indent');
-}
+  let optionsElementId = '#order-blocks-options-' + uuid;
+  let dropzoneElementId = '#order-blocks-dropzone-' + uuid;
 
-function check_block(event, ui) {
-  var block_parent_name = $(ui.item.parent()[0]).attr('name');
-  var block_destination_name = event.target.getAttribute('name');
-  if (block_parent_name !== block_destination_name) {
-    return false;
-  }
-  return true;
-}
+  function setAnswer(event) {
+    var domObjs = $(dropzoneElementId).children();
```

Comment:
Can we give this a more meaningful name than `domObjs`?

Reply:
sure